methods.go

```diff
@@ -83,3 +70,34 @@ func EnableColor(flag bool) {
 func ListLoggersNames() []string {
 	return []string{prettylogger.LoggerName, zaplogger.LoggerName, nonelogger.LoggerName}
 }
+
+// InitOtel configures OpenTelemetry to export data to OTEL_COLLECTOR_SVC using uptrace collector.
+// You have to set the env variable OTEL_COLLECTOR_SVC to enable otel.
+// It is required to call ShutdownOtel on the context at the end of the main.
+func InitOtel(log helpers.ILogger, service, version string) context.Context {
+	ctx := context.Background()
+	if svc, ok := os.LookupEnv("OTEL_COLLECTOR_SVC"); ok {
+		var attrs []attribute.KeyValue
+		if id, ok := os.LookupEnv("ACCOUNT_ID"); ok {
+			attrs = append(attrs, attribute.String("account.id", id))
+		}
+		dsn := url.URL{
+			Scheme: "http",
+			User:   url.User("t"),
+			Host:   svc,
+			Path:   "1",
```

Comment:
question: why do we use these exact values for `User` and `Path`?

Is there any special meaning to `User: url.User("t")` and `Path: "1"`?

Reply:
This is because we're using the uptrace client (see reason elsewhere) and they expect an url formatted like "https://<token>@uptrace.dev/<project_id>".
In practice this adds a special header to the gRPC data to tell uptrace how to handle them.
However in our case we are sending to an otel Collector so we don't care about them.
Eventually if our user wants to send to uptrace, he can set this header directly inside the collector config like [here](https://uptrace.dev/opentelemetry/collector.html#configuration).

Reply:
What happens if we omit `User` and `Path`? Will the Uptrace client will return an error and complain?

The reason I’m asking is because we would like to be agnostic of the telemetry backend. For example, if `go-logger` clients would like to use it with something other than Uptrace which requires different `User` and `Path`, would they be able to do it? Or if a user would like to use a managed Telemetry instance outside of their in-cluster components.

Maybe we should accept the service URL as a `url.URL` and adjust it if `User` and `Path` are empty?